app/test_main.py

```diff
@@ -1,3 +1,34 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected",
+        [
+            (-1, -1, [0, 0]),
+            (14, 1, [0, 0]),
```

Comment:
in this case, you test one value only for the cat but not for the dog, the same in other cases. You should check all age boundaries for both of them

Reply:
and what about 29 for a dog?)

Reply:
fixed